src/modules/content-tree/content.tree.module.js

```diff
@@ -2,20 +2,79 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import List from './components/list/list.component';
 
+const CLASS_IS_TREE_RESIZING = 'ez-is-tree-resizing';
+
 export default class ContentTreeModule extends Component {
     constructor(props) {
         super(props);
 
+        this.changeContainerWidth = this.changeContainerWidth.bind(this);
+        this.addWidthChangeListener = this.addWidthChangeListener.bind(this);
+        this.removeWidthChangeListener = this.removeWidthChangeListener.bind(this);
+        this._refTreeContainer = React.createRef();
+
         this.state = {
             selectedLocationId: props.selectedLocationId,
             locations: props.preloadedLocations,
```

Comment:
I don't like idea of mixing logic responsible for resizing with selecting location etc. In my opinion the way to go is extracting logic responsible for resizing to another component.

Reply:
In a context of the content tree I disagree with your approach. I would create another component only when it will be reused in other places. Currently, there's no need to do it.
There's no need to create another class/component just for it. It simplifies the way the code works without extending a bundle size (in bytes) and all the logic for the content tree module is one place.